compliance/solr/pom.xml

```diff
@@ -10,6 +10,10 @@
 	<name>RDF4J: Solr Sail Tests</name>
 	<description>Tests for Solr Sail.</description>
 	<properties>
+		<!-- FIXME: Delete; Jetty 10.x requires Solr 9.x -->
+		<jetty.version>9.4.50.v20221201</jetty.version>
```

Comment:
I'm still not sure what this means for us.

Reply:
It means what it says in the comment. 😉 Some of the Solr dependencies used for testing are hard-locked to Jetty 9. I hope we can fix this by upgrading Solr before rdf4j 5 is released. Or even better: find a way to avoid being dictated by legacy transitive dependencies.